lib/themes/dosomething/paraneue_dosomething/includes/helpers.inc

```diff
@@ -221,13 +221,14 @@ function paraneue_dosomething_get_search_gallery($results) {
  *   The source query string to append to the link.
  */
 function paraneue_dosomething_get_node_gallery_item($nid, $source = NULL) {
-  $classes_array = array();
+  $classes_array = [];
 
   $node = node_load($nid);
-  $link = drupal_get_path_alias('node/' . $nid);
+  $url_options = [];
   if ($source) {
-    $link .= '?source=' . $source;
+    $options['query']['source'] = $source;
```

Comment:
Please note that `$url_options` is not the same as `$options`.  `$options` is created on spot an never used. `$url_options`, on the other hand, stays empty.